app/src/main/java/fr/motoconnect/ui/screen/authentication/SignUpScreen.kt

```diff
@@ -100,17 +120,58 @@ fun SignUpScreen(
             ),
             keyboardActions = KeyboardActions(
                 onDone = {
-                    authenticationViewModel.signUp(email.value, password.value, displayName.value)
+                    if(imageUri == null) {
+                        imageUri = Uri.parse("android.resource://fr.motoconnect/${R.drawable.moto}")
+                        authenticationViewModel.signUp(email.value, password.value, displayName.value, imageUri!!)
+                    }else{
+                        authenticationViewModel.signUp(email.value, password.value, displayName.value, imageUri!!)
+                    }
                 }
             ),
             onValueChange = { password.value = it },
         )
 
+        Spacer(modifier = Modifier.height(20.dp))
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(0.dp, 0.dp, 0.dp, 15.dp),
+            horizontalArrangement = Arrangement.Center
+        ){
+            imageUri?.let {
+                Image(
+                    painter = rememberAsyncImagePainter(model = imageUri),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .clip(CircleShape)
+                        .size(100.dp, 100.dp)
+                )
+            }
+            Spacer(modifier = Modifier.width(10.dp))
```

Comment:
Tu peux utiliser le spacedBy sur le modifier de row pour faire ça

Reply:
Je rate peut etre un truc mais quand j'essaie de rajouter un spacedBy dans le modifier comme ceci : 
```suggestion
                modifier = Modifier
                .fillMaxWidth()
                .padding(0.dp, 0.dp, 0.dp, 15.dp)
                .spacedBy()
```
Je me tape une "unresolved reference" error

Reply:
my bad, c'est pas sur le modifier, mais sur horizontalArrangement https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/Arrangement#spacedBy(androidx.compose.ui.unit.Dp,androidx.compose.ui.Alignment.Horizontal)